browser-test/src/support/config.ts

```diff
@@ -11,5 +11,5 @@ export const {
   // This value was extracted from the browser test env via:
   //
   // python -c "import requests;print(requests.Session().get('http://localhost:9999').cookies.get_dict()['PLAY_SESSION'])"
-  FROZEN_PLAY_SESSION_COOKIE_VALUE = 'eyJhbGciOiJIUzI1NiJ9.eyJkYXRhIjp7InNlc3Npb25JZCI6IjE4Y2JkYTRhLWFjYmEtNDY5OS05ZTM0LWY2ZGI0OWQxZGNhYSIsInBhYzRqIjoiZnROVURWZFJJZWJrdkFldndadmdMN0RoZ0d4a0dPNUpxMTJkVUE0Sjl4RHQ5cDFnN0J4dXZ1YkUzQUVzdzNCQVQwNHZ0bGNLQ2plWjRHV1lTK295cDU5OUE0TGR4WEJvcnVLZ01tKzg2S3J3L0FmV3c5MkJYbjRRU2dzSjRnQ3VCWkJSMVRXaENVWUNBZStlWjd3d2h1R2g2MTlqcHpqWjlSNXZEbU04Z0Vod25samdQNml1VENsRVZ6OGZXRWo0QTdkWHMyWUxXY2dUSWp4RnplOEh3WjQ0TmVRN2R1d3U4d1gvNEZIRitqTFlMazdGdUtMbGtXVWs5L0o2QTlIejRma3ZRaDV1a1YxYktBc2tscmo0aVloSTg2a2Fodks0TTJlUFBaQXVCZm1SY1ZFSCtySXozZWdScFpZRFBNWFhpTDBpV0Q3STF4RElsUDZUSlFIcCtYL3k0QjlNWGJjT1p6elBjeVVwR0JqVWNwWWhQUjVZSzA4bVFhRk1BUkJNWkNXUWthNWJaS2JNR3NVQUZmM0kydjdDdjdtOUphTjI4UFRvNW12Nk8vSDIzTWRSNk0wZkd1aEVwa0pJZFpYUi9JQXRRWVFiMXo4NWNqdi9uNEFvNzBVSk50NS84a0pRdWZMR1lFUEh4ay8xNkVQc20yNlIvMGp4cWVEQWRBK2NZM0ZhZ0NzZjhxYVdPaURLblpkNFNObkQvMTk3ejdIWTFEbGdwcVdwNENzc1V6cWJpSjhZV2lKOGVPMXN5UEU2MUFBd3ZyN05abzIxOXlQTzJEVlRrZkhrNU5senQ4eGNEaXlaemQrUGsxSVVnVEt6QUFXVzJjb3IwMUJXTjU3VHV2eHYyYjFMRXgyUnNCc05ERFlJMU94VXprbmgvMGxjT1hDdFgxdy82TFVDa3ZlOUFQQ0NnZTNGa2NmcFgxWEdpS0pqNk5TYXp3RkNCZEVHblQxemNPeDM5SWtaTU04d010ZHMiLCJjc3JmVG9rZW4iOiIyNzJkYjEyMWFjMDc2MGEwNzRiNjg4N2EyYThhMjE4ODBhNjBmN2VlLTE3MDE0NTgyMjQxNDYtY2MxMzI4ZWFlMTkxNjI4NTJjMDE0MzhhIn0sIm5iZiI6MTcwMTQ1ODIyNCwiaWF0IjoxNzAxNDU4MjI0fQ.ifm74kjDkb-eEUw2nLJY9f3TaQe2QHHztW_1M1W1-zM',
+  FROZEN_PLAY_SESSION_COOKIE_VALUE = 'eyJhbGciOiJIUzI1NiJ9.eyJkYXRhIjp7InNlc3Npb25JZCI6IjU1MTMwZDYyLTA3M2QtNDQwNi05MGI5LWJkOWU0NjNhM2U5NyIsInBhYzRqIjoiWDFBM1Z6YnVUOVJLZ24xNmZ1QXNCT25DY0swb1BwcFA3NTlUMzR1NXAvTndMNk5qS2ovL0dyZDhaUUN5UU1FcUhvbXJsTEV0VXJPYnFZRlp4dXlOck8wb3Boayt2eFhYY3dzWis4YUZxeHpLWnkvdlBtcnIvd3FVT21SRjhPNm1SZzBqQ2hZRjB4ZFdBa0lQNGlnNUhkZW5tZlptSHcyYmg2N2JVWnB1MGlMREFTQmdGbHhEaDdDamdHRXNGZnRGYmVSRWtoVjFUdzZacWxDZTM3MFR3dXg2NmNmSWp2c092NHN4Q0FMVG4wVGVxbUR4K2tpM29ibWQ1TFNyLzhhb1RBY2kzTGFoOERLczQrVWxKUHpKOGZLVHZkTzFXS0kwSjZ0MjdBbGx2T2N1ZkxtRVJXM0Y2dnpVU0l5bTVMa1ZSS01hUlFWOG9Da1JESlQrV09xQk1qTDBJdXJ3SUFrQ3UyZCtqUmxTZklqTnBISWVvcDFrUTNEa1BMejM0QVNvNEVNWml1VUhER3lWZkZhWW5aK2YyVWZESkxWeWs2ZmhRNjBZTWlXRG5OMUtHSGlON3UzcVIzZ3grSVZ3OWtONGVQMm5SbmRuTWluQnZnem9qUlE3VXlFcE55eHUvQnBDQTlBRXlLcENpTkxudmJvTmh4UWw2d0MxSmN1Z0NHK1lwRFkycEhOcTl3UXNRZktRRDQwdlo5eGlPRXhHV0tzdkFQTGZ6dllwVFhFMTFjRkdJVTJ4cUgrZW9yelNGMjFqa2laSVlqQmM0N3NpS25HVWordHlVbUh3K0pLdnlqUDhiajB2WUhiN0kzRUJkbERhaTVHTnRNK3BDU0l1Zkg0Mkg1OEtOSkRiMWdKdzRocXl4QzBqL0ZMQmtibUR6SXNwTW16WEY4U3lDT0FvV0o4bjZVZFpyOG52MjVleUVDMDR5VlF1SlVBR3BwZzdnYktqdXd0VnBmN0tlNXowaExiMWpyT05yTFFBQWFSckYzVTZRSm0vVEE9PSIsImNzcmZUb2tlbiI6Ijc2ZWNlNWZlZTRkNzgwZDEzZTZjZmJiODAzMWI3YTFmMWZjODM2Y2YtMTcwNTcxOTg2NjUzNy01MDJjNWZiZTlmNmFhMmRhYzhhMGE1NDQifSwibmJmIjoxNzA1NzE5ODY2LCJpYXQiOjE3MDU3MTk4NjZ9.WYG9_nsRo4Bjev7IG3rK4JJJfWWBJGhkT1wZDecKmVs',
```

Comment:
**question:** What's the impact of changing this?

Reply:
Tom added it for some tests. It is changing because the secret key changed.

Reply:
Oh right! Makes sense.